referencemodels/src/main/java/org/openehr/referencemodels/BuiltinReferenceModels.java

```diff
@@ -59,20 +59,20 @@ public static BmmRepository getBmmRepository() {
                 "bmm/FHIR/DSTU/BMM/hl7_fhir_resourceses_dstu.bmm",
                 "bmm/openEHR/components/RM/Release-1.0.3/openehr_demographic_103.bmm",
                 "bmm/openEHR/components/RM/Release-1.0.4/openehr_rm_104.bmm",
-                "bmm/ISO_13606/2008/BMM/cen_ts14796_0.90.bmm",
+                // "bmm/ISO_13606/2008/BMM/cen_ts14796_0.90.bmm",
```

Comment:
Some of these things are occassionally used, such as CIMI. May need to update these instead of removing them.